.github/workflows/ci-for-default-ns.yaml

```
@@ -67,7 +66,15 @@ jobs:
           kubectl rook_ceph rook status
           kubectl rook_ceph rook status all
           kubectl rook_ceph rook status cephobjectstores
+          # use of sleep is not incouraged but here we need sleep between scaling down the osd and purging the osd
```

let's just explain why we're sleeping, no need for explaining why we wish we didn't have to ;) 
```suggestion
to allow time for reconcile, sleep before listing the pools
```